GeneratedRegexPatterns/LowConfidenceSecurityModels.json

```diff
@@ -8,5 +8,40 @@
       "ret="
     ],
     "DetectionMetadata": "LowConfidence"
+  },
+  {
+    "Pattern": "^(?i)[a-z0-9.=\\-:[_@\\/*\\]+?]{32}$",
```

Comment:
Can we change the pattern to `(?i)^[a-z0-9.=\\-:[_@\\/*\\]+?]{32}$`?

I got `re.error: global flags not at the start of the expression` while testing